Changing the link which forwards to the yaml config of the collector (#113)
hashicorp/terraform-docs-agents: website/docs/cloud-docs/agents/telemetry.mdx

### Configuring the collector

The OpenTelemetry collector takes configuration in the form of a YAML file. Refer to the [default YAML configuration file](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/config.yaml) as a starting point for writing your own configuration.
may be useful to refer to the default YAML configuration file as a starting

> point for your own configuration. Its source may be found

Reviewer: ```suggestion
```

> [here](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/config.yaml).

Reviewer: ```suggestion
```

> The OpenTelemetry collector takes configuration in the form of a YAML file. It

Reviewer: Suggesting rewriting the following sentences to meet our style guide -> if you like this you'd remove the following lines. Thank you so much!
```suggestion
The OpenTelemetry collector takes configuration in the form of a YAML file. Refer to the [default YAML configuration file](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/config.yaml) as a starting point for writing your own configuration.
```
